features/authentication/authentication.api.ts

```diff
@@ -78,36 +76,37 @@ export const createUser = async (args: {
     // Validate request payload
     const validationResult = createUserRequestSchema.safeParse(requestPayload);
     if (!validationResult.success) {
-      logger.error(
-        `[createUser] Request validation failed:
-        ${JSON.stringify(validationResult.error.errors, null, 2)}
-        Payload: ${JSON.stringify(requestPayload, null, 2)}`
+      const error = new Error(
+        `Invalid request data: ${validationResult.error.message}`
       );
-      throw new Error("Invalid request data");
+      captureError(error);
+      throw error;
```

Comment:
I went too fast earlier. There's another problem here. This will log too many errors. Like first we'll log it here and then since we throw the parent might log too. We always prefer just throwing and letting the parent capture the error and decide what to do with it IMO